config/prod/bgrande-nextflow-service-account.yaml

```diff
@@ -0,0 +1,14 @@
+# Provision a service account for AWS API access keys
+template_path: "remote/service-account.yaml"
+stack_name: "bgrande-nextflow-service-account"
+parameters:
+  Department: "CompOnc"
```

Comment:
there are not input parameters to `service-account.yaml` template.  You probably want to move Department, Project, and OwnerEmail to a `stack_tags:` dict.